src/App/Pages/Vault/CipherDetailsPageViewModel.cs

```diff
@@ -405,12 +405,6 @@ public async Task<bool> RestoreAsync()
                     AppResources.InternetConnectionRequiredTitle);
                 return false;
             }
-            var confirmed = await _platformUtilsService.ShowDialogAsync(AppResources.DoYouReallyWantToRestoreCipher,
-                null, AppResources.Yes, AppResources.Cancel);
```

Comment:
⛏️ Given that `AppResources.DoYouReallyWantToRestoreCipher` is no longer used, can you remove it from `AppResources` as well?